docs/galaxy_releases.rst

```diff
@@ -0,0 +1,218 @@
+  readme: README.md
+
+  # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+  # @nicks:irc/im.site#channel'
+  authors:
+    - "Author1"
+    - "Author2 (https://author2.example.com)"
+    - "Author3 <author3@example.com>"
+
+
+  ### OPTIONAL but strongly recommended
+
+  # A short summary description of the collection
+  description: Description of what this collection has
+
+  # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+  # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+   license:
+     - GPL-2.0-or-later
+
+  # The path to the license file for the collection. This path is relative to the root of the collection. This key is
+  # mutually exclusive with 'license'
+  # license_file: ''
+
+  # A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+  # requirements as 'namespace' and 'name'
+  tags: []
+
+  # Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+  # collection label 'namespace.name'. The value is a version range
+  # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+  # range specifiers can be set and are separated by ','
+  dependencies: {}
+
+  # The URL of the originating SCM repository
+  repository: https://github.com/my_org/my_collection
+
+  # The URL to any online docs
+  documentation: https://github.com/my_org/my_collection
+
+  # The URL to the homepage of the collection/project
+  homepage: https://github.com/my_org/my_collection
+
+  # The URL to the collection issue tracker
+  issues: https://github.com/my_org/my_collection/issues
+
+
+Configure GitHub Workflow
+-------------------------
+
+To be able to execute the Ansible playbook we need to configure the GitHub
+Actions Workflow. To do this we will need to create the
+:code:`.github/workflows/` folder if it does not already exist. Then create a
+new file :code:`release.yml` and it should look something like this.
+
+.. code-block:: yaml
+
+  name: "release"
+  on:
+  release:
+    types:
+      - created
+  jobs:
+  release:
+    runs-on: ubuntu-18.04
+    env:
+      ANSIBLE_GALAXY_TOKEN: ${{ secrets.ANSIBLE_GALAXY_TOKEN }}
+      ANSIBLE_FORCE_COLOR: 1
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ansible
+
+      - name: Run role test
+        run: >-
+          ansible-playbook -i 'localhost,' build/galaxy_deploy.yml -e "github_tag=${{ github.ref }}"
+
```

Comment:
Is there a reason to have both a playbook and an action? Why not put everything into an action or create different instructions for people who want to do things differently

Reply:
The playbook does the templating and processing of the values. What action do you see here?

Reply:
I mean, the GitHub action. Although templating might be less easy with a raw action.

Reply:
Let's see where the discussion ends up, maybe that ends up being where we take this now that people want all these other steps in place.

Reply:
Yeah, it could be a great idea to actually create our own action. But I'd not mix build&publish, because one should inject some testing in between, so it'd be two actions.

Reply:
What are the proposed steps then?

Reply:
If you're up to writing a few lines of bash and a dockerfile, you could create an action or two. The question is whether it's overkill for this task, though.